packages/styles/src/themes/bootstrap/_overrides.scss

```diff
@@ -1,5 +1,5 @@
-@import './overrides-variables';
+// DEPRECATED
+// this file is deprecated and will be removed in the future
+// instead use '/src/themes/bootstrap/_core.scss'
 
-@import 'bootstrap/scss/functions';
-@import 'bootstrap/scss/variables';
-@import 'bootstrap/scss/mixins';
+@use './core';
```

Comment:
Delete this file if it's no longer used. I don't expect anybody else to be affected from this change so it's not a breaking change.